### ISSUE TITLE
migration of outline names

### ISSUE THREAD

**Issue by eroux** [BuddhistDigitalResourceCenter/xmltoldmigration#23]
Outline nodes may have names and may have titles. The first encountered name (if any) is transformed as a `skos:prefLabel`, and so is the first title if there is no name. Titles are also transformed into regular work titles like

```ttl
    :workTitle          [ a                   :WorkBibliographicalTitle ;
                          rdfs:label          "gleng gzhi/"@bo-x-ewts
                        ] ;
```

but names don't. This makes the queries a little bit difficult: for the BFA data I want to get all the titles and names in a unique way... I can easily do some code for that by checking if the prefLabel is already in the list of titles and if not add it, but I think it may reveal a defect in the modeling... Maybe we should transform names into `:workPartLabel`? I suspect the names of the outline nodes denote some sort of place of the work in the outline hierarchy (like "second part" or so), and I think this information may be valuable enough to have its own field? wdyt @xristy?

**Comment by xristy** [comment]
The `<o:name/>` elements are used, as I think you indicate, for hierarchical portions of the outline that organize "true" Works (like a bunch of sutras) into a group such as the "Sutra Basket" and so on.

I think I understand your issue and want to clarify whether you would have both a `skos:prefLabel` and `:workPartLabel` with the same content or?

**Comment by eroux** [comment]
yes I would have both, so that we don't loose track that we do have a `:workPartLabel`

**Comment by xristy** [comment]
ooops! left off the fix ... Fixed in [commit 3467b4e9](https://github.com/BuddhistDigitalResourceCenter/owl-schema/commit/3467b4e920a0d31b7adb7360cd5e0bfe06b17d58)
